src/server/index.js

```diff
@@ -1,15 +1,20 @@
 'use strict';
 
 const express = require('express');
-const app = express();
 const path = require('path');
+
 const PORT = process.env.PORT || 3000;
+const app = express();
+
+app.use(express.static(path.resolve(__dirname, '../../dist')));
 
 app.get('/', (req, res) => {
```

Comment:
you dont need to define this endpoint. 
1, (unnecessary) You have already set the static folder so html files will be served automatically from the dist folder
2, (buggy) In this endpoint you serves the template file from the "src" folder instead of the build index.html from the "dist"